Assets/Scripts/Scenes/InGame/Player/Item/ItemEffectScript/BandageEffect.cs

```diff
@@ -8,54 +8,70 @@ namespace Scenes.Ingame.Player
 {
     public class BandageEffect : ItemEffect
     {
-        private float _timer;
+        private float _startTime;
         private bool _stopCoroutineBool = false;
+        private int speed;
         private void Start()
         {
             base.SetUp();
         }
 
         public override void OnPickUp()
         {
-            //��юg�p���Ƀ_���[�W���󂯂����т̎g�p������
-            ownerPlayerStatus.OnPlayerHealthChange
-                .Where(x => ownerPlayerStatus.lastHP > x)
-                .Subscribe(_ => _stopCoroutineBool = true)
-                .AddTo(this);
+            //�U����������Ƃ�������Bool��True�ɂȂ����Ƃ��ɃA�C�e���g�p�𒆒f
+            ownerPlayerStatus.OnEnemyAttackedMe
+                .Skip(1)//�������̎��͖���
+                .Subscribe(_ => _stopCoroutineBool = true).AddTo(this);
         }
 
         public override void Effect()
         {
-            StartCoroutine(UseBandage());
-            //ownerPlayerStatus.ChangeBleedingBool(false);
+            if (ownerPlayerStatus.nowBleedingValue)
+                StartCoroutine(UseBandage());
+            else
+                Debug.Log("�o����Ԃł͂Ȃ��̂Ŏg�p���܂���ł����B");
         }
 
         public IEnumerator UseBandage()
         {
             Debug.Log("��юg��");
+            
             //�A�C�e���d�l����ɃX�e�[�^�X�ύX���s��
-            int speed = ownerPlayerStatus.nowPlayerSpeed;
+            speed = ownerPlayerStatus.nowPlayerSpeed;
             ownerPlayerStatus.ChangeSpeed(ownerPlayerStatus.nowPlayerSpeed / 2);
```

Comment:
```suggestion
            ownerPlayerStatus.UseItem(true);
            ownerPlayerStatus.ChangeSpeed();
``` 
別のコメントで書いた修正案の場合の変更箇所
以下while分も同様に変更すると良いと思います（useItemの引数はfalse）